src/index.js

```diff
@@ -0,0 +1,65 @@
+import './style.css';
+
+const todoTasks = [
+  {
+    description: 'Read the documentation',
+    completed: false,
+    index: 1,
+  },
+  {
+    description: 'Create the Repository',
+    completed: false,
+    index: 2,
+  },
+  {
+    description: 'Set the Linters',
+    completed: false,
+    index: 3,
+  },
+];
+
+const listDiv = document.getElementById('list');
+
+// Function to add a new task to the todoTasks array and display it on the screen
+const addTask = (description, index) => {
+  const task = {
+    description,
+    completed: false,
+    index,
+  };
+
+  todoTasks.push(task);
+
+  const singleTask = document.createElement('li');
+  singleTask.className = 'single-task';
+  singleTask.id = task.description;
+
+  const checkbox = document.createElement('input');
+  checkbox.type = 'checkbox';
+  checkbox.id = task.index;
+  checkbox.name = task.completed;
+
+  const taskDescription = document.createElement('p');
+  taskDescription.innerText = task.description;
+
+  singleTask.appendChild(checkbox);
+  singleTask.appendChild(taskDescription);
+  listDiv.appendChild(singleTask);
+};
```

Comment:
- [ ] Nice work populating the HTML template for each task :clap: However it is missing the icons at the right side of the task list. Kindly make sure your list is a clone of the part of the minimalist project captured in the [video](https://www.youtube.com/watch?v=AcUd-_Yjjqg) provided.
### Your design :point_down: 
![Screenshot from 2023-07-18 14-55-34](https://github.com/AliRezaBmeDu/todo-list/assets/91142432/69e4739f-2330-4b6d-a180-a35af8974b7a)
### What it is supposed to look like :point_down: 
![Screenshot from 2023-07-18 14-54-33](https://github.com/AliRezaBmeDu/todo-list/assets/91142432/f8cb38b6-3079-48ee-8521-65243a22afba)